packaging/cockpit-machines.spec.in

```diff
@@ -35,7 +35,9 @@ Requires: cockpit-bridge >= 215
 %if 0%{?suse_version}
 Requires: libvirt-daemon-qemu
 %else
-Requires: libvirt-daemon-kvm
+Requires: libvirt-daemon-driver-qemu
```

Comment:
i am afraid we need at least the network and the storage libvirt core packages here, otherwise virt-install will fail to create the default pool/network. (as seen now on the tasks container anaconda webui tests) did you try it out without?

Reply:
Hmm, `virt-install` ought to depend on these then, but doesn't. `virt-install` itself is only recommended, but we can do something like

    Requires: (libvirt-daemon-driver-storage-core if virt-install)

(and the same for network). WDYT?

Reply:
Done, thanks for pointing out!